ngraph/core/include/openvino/pass/make_stateful.hpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
2018 is excessive

Reply:
Actually this is ok from legal perspective, but still it look weird in new files.

Reply:
2018 is ok, we have already consulted with legal about this.
I guess it's easier to replace them when they are the same everywhere.

Reply:
@itikhono Maybe you are right, but we have a lot of such copyrights in other files:
```
// Copyright (C) 2021 Intel Corporation
``